src/01.UI-Elements/ListItem.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+ListItem.propTypes = {
+  titleListItem: PropTypes.string.isRequired,
+  isCheckmarked: PropTypes.bool,
+}
+
+export default function ListItem({ titleListItem, isCheckmarked }) {
+  return (
+    <ElementStyled>
+      <CheckboxStyled type="checkbox" checked={isCheckmarked} />
+      <TitleStyled>{titleListItem}</TitleStyled>
+    </ElementStyled>
+  )
+}
+
+const ElementStyled = styled.label`
```

Comment:
A label is a label...

Reply:
I wanted to indicate that this is the most outer style the user will see...